support-frontend/assets/pages/supporter-plus-landing/components/__snapshots__/paymentTsAndCs.test.tsx.snap

```diff
@@ -783,7 +783,7 @@ exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product TierThree
         .
          
         <p
-          class="css-15t9qdk-FooterTsAndCs"
+          class="css-1echgku"
```

Comment:
Maybe my Mac but thsi looks updated again to me eg `pnpm test paymentTsAndCs.test` gave me this  -> 
![image](https://github.com/user-attachments/assets/c1592a08-a961-4071-9c39-023986fdda4d)

Reply:
Did you run `pnpm install` first to make sure you're on the version of `@emotion/react` upgraded in this PR?